### ISSUE TITLE
Couldn't create directory for SharedPreferences file /data/data/net.domain.app/shared_prefs/disk_entries_list_images_-42367832.xml

### ISSUE THREAD

**Issue by eldk** [facebook/fresco#1378]
Hello,

This error occure when using last fresco master https://github.com/facebook/fresco/commit/f3f23666145079fca25aeddb652b5ae2c5adb5e2 to build my app.

Switch back to  commit level, https://github.com/facebook/fresco/commit/4cd620a09ae28f71d89cd2305c8a17357e4fe969, this error is not there. (using this branch : https://github.com/eldk/fresco/tree/downsample+libwebp0.5.0)

Thanks, 

Eric

**Comment by kirwan** [comment]
Did this problem go away? That's a new shared preference but I wouldn't expect it to fail to create the file.

**Comment by eldk** [comment]
Hello,

I had closed it, because it can't be reproduced after a device memory clean.

As I removed the app from device before to reinstall it, Google Play Services was reinstalled and full fill available device memory.

Remove Google Play Services and free memory make it OK. I will check again, and check why Google Play Services is coming back.

So maybe a check to available memory before to write this file is needed. 

This error make Fresco crashed on every picture prefetch, display, . So it's unusable.

Thanks,

Eric
